website/client/components/groups/group.vue

```diff
@@ -105,7 +105,7 @@
       .section(v-if="sections.challenges")
         group-challenges(:groupId='searchId')
     div.text-center
-      button.btn.btn-danger(v-if='isMember', @click='clickLeave()') {{ $t('leave') }}
```

Comment:
I think this string is only used there so we can remove it

Reply:
It's still used in buttons on the guild list pages:
![img](https://screenshotscdn.firefoxusercontent.com/images/a46bed9f-cc3f-46cf-80b4-a7e7b6b35259.png)